BookingsApi/BookingsApi/Services/UpdateHearingService.cs

```diff
@@ -44,7 +44,7 @@ public async Task UpdateParticipantsV1(UpdateHearingParticipantsRequest request,
         public async Task UpdateParticipantsV2(UpdateHearingParticipantsRequestV2 request, 
             VideoHearing hearing, List<HearingRole> hearingRoles)
         {
-            await _hearingParticipantService.UpdateParticipantsV2(request, hearing, hearingRoles);
+            await _hearingParticipantService.UpdateParticipantsV2(request, hearing, hearingRoles, false);
```

Comment:
can you specify the argument name like

`await _hearingParticipantService.UpdateParticipantsV2(request, hearing, hearingRoles, sendNotification: false);`

same with the other 2 lines below

Reply:
done